### ISSUE TITLE
%matplotlib inline interaction?

### ISSUE THREAD

**Issue by nicolaskruchten** [plotly/plotly_express#13]
> I did notice one thing: if I put %matplotlib inline to also use mpl plots (cause legacy stuff) then the px stops rendering - it makes a blank space in the notebook of about the size the figure would be.

From https://www.reddit.com/r/Python/comments/b3doya/introducing_plotly_express/ej0p664/

**Comment by khalido** [comment]
Hi, I made that reddit post linked to above, just clarifying that it doesn't seem to have anything to do with using `%matplotlib inline%, I've getting that behaviour in new, blank notebooks as well.

So its the as [issue 17](https://github.com/plotly/plotly_express/issues/17).

I'm using Anaconda 2018.12, installed plotly_express using `pip install plotly_express`.

I'm testing this out in a new conda env with just pip installed and then doing `pip install plotly_express`.

it imports fine:

```
import plotly_express as px
print(px.data.iris.__doc__)
iris = px.data.iris()```

prints out info and I get a iris data frame, but doing this:

`px.scatter(iris, x="sepal_width", y="sepal_length")` or

```
fig = px.scatter(iris, x="sepal_width", y="sepal_length")
fig
```
gives me a blank area the size of the plot. 

It seems to be in offline mode as `fig.offline_initialized` returns True.

**Comment by nicolaskruchten** [comment]
OK, we'll take a look! which versions of Python and JupyterLab are you using?

**Comment by nicolaskruchten** [comment]
I've updated the README to note that the `plotly-extension` needs to be installed if using JupyterLab via `jupyter labextension install @jupyterlab/plotly-extension` ... This resolved #17 apparently.
